CONTRIBUTING.md

```diff
@@ -374,3 +380,42 @@ class here.
 
 To add a new server specification, create a new file in this directory and add the file name to the `all-servers.json` 
 file.
+
+
+### Command style guide
+Below are guidelines for adding or modifying commands. The goal is to have a consistent presentation across commands.
+These are not hard rules, but should apply to most commands. Always choose better usability over following a rule here.
+
+#### Options instead of parameters
+Use options instead of parameters, wherever possible. 
+e.g. `terra workspace add-user --email=user@gmail.com --role=READER`
+instead of `terra workspace add-user user@gmail.com READER`.
+
+This makes it easier to maintain backwards compatibility when adding new arguments. It also makes it easier to read
+commands with multiple arguments, without having to remember the order.
+
+The exception to this rule are the `config get-value` and `config set` commands. These will only ever take one argument
+and use a parameter instead of an option.
```

Comment:
no strong opinion here, but if there's not a compelling reason to have a single exception, maybe we just shouldn't have an exception. If there's a general rule about when we think using parameters is preferable to using arguments, might be worth trying to capture that here with config get-value and set as examples. (Doesn't `set` kind of take 1 parameter, but get-value takes 0, but get-value can sometimes have a json format argument?)

aside not necessarily this PR: consider making config `get-value` and `set` symmetrical, e.g. either `get` & `set` or `get-value` & `set-value`.

Reply:
Ah you're right that `config get-value` doesn't take any parameters, just the `--format` option. The only exception is `config set` that takes a parameter for the config property value. I thought about changing it to use an option instead e.g. `--value`, but that seems uninformative. I don't feel strongly either way though, so happy to be talked out of it.

RE `get-value` and `set`: I did that to be consistent with `gcloud` which uses these same command names. Maybe that's not a very generic choice though, so we don't gain much by copying them. Thoughts?

Reply:
I don't feel particularly strongly either, sorry.
Could do `--get=key` and `--set=key --value=value`, but I don't know that's much improvement.

Didn't know it came from gcloud, but I'd probably lean towards not copying them for this. Maybe that'll upset some gcloud frequent users.